Jenkinsfile

```diff
@@ -291,13 +291,13 @@ def generateStages(Map args = [:]) {
 def cloud(Map args = [:]) {
   withGithubNotify(context: args.context) {
     withNode(labels: args.label, forceWorkspace: true){
-      startCloudTestEnv(name: args.directory, dirs: args.dirs)
+      startCloudTestEnv(name: args.directory, dirs: args.dirs, cloudAWS: args.cloudAWS)
     }
-    withCloudTestEnv() {
+    withCloudTestEnv(args) {
       try {
         target(context: args.context, command: args.command, directory: args.directory, label: args.label, withModule: args.withModule, isMage: true, id: args.id)
       } finally {
-        terraformCleanup(name: args.directory, dir: args.directory)
+        terraformCleanup(name: args.directory, dir: args.directory, cloudAWS: args.cloudAWS)
```

Comment:
question: why adding `cloudAWS: args.cloudAWS` is needed for `terraformCleanup`?
should we add as well to `terraformApply`? (https://github.com/elastic/beats/blob/master/Jenkinsfile#L922)

Reply:
`terraformClean` calls `withCloudTestEnv` internally.

Good catch, I missed `terraformApply`.

OTOH, I'm still working on this, since I wanna double check the changeSet when condition in the `Jenkinsfile.yml` works in real :)

Reply:
Actually this is not needed, since `terraformApply` it's called only by `startCloudTestEnv` and this method is only called by `cloud`.

In other words, those two methods don't need to do anything since their callers already configured the `cloudAWS: args.cloudAWS`